src/overlays/actors/ovl_Obj_Mure/z_obj_mure.c

```diff
@@ -172,7 +172,7 @@ void ObjMure_SpawnActors1(ObjMure* this, GlobalContext* globalCtx) {
     for (i = 0; i < maxChildren; i++) {
         ObjMure_GetSpawnPos(&spawnPos, &actor->world.pos, this->ptn, i);
         this->children[i] = Actor_SpawnAsChildAndCutscene(
-            &globalCtx->actorCtx, globalCtx, sSpawnActorIds[this->type], spawnPos.x, spawnPos.y, spawnPos.z,
+            &globalCtx2->actorCtx, globalCtx, sSpawnActorIds[this->type], spawnPos.x, spawnPos.y, spawnPos.z,
```

Comment:
I assume this is required.

Reply:
Yeah, though the second `globalCtx` is free to be either. For consistency I think both being `globalCtx2` is best.

Reply:
On the contrary, this proves that this function takes state as an argument.

Reply:
Ah, that makes sense. Reverted it back.